R/calc_fit_mod.R

```diff
@@ -0,0 +1,43 @@
+#' @title Calculates fitted and residual values for modeled outputs
+#' @description
+#'  `calc_fit_mod()` takes antibody kinetic parameter estimates and calculates
+#'  fitted and residual values. Fitted values correspond to the estimated assay
+#'  value (ex. ELISA units etc.) at time since infection (TSI). Residual values
+#'  are calculate as the difference between fitted and observed values.
+#' @param input_dat A [] dataframe of modeled antibody kinetic parameter values.
+#' @param original_data A [] data frame of the original input dataset.
```

Comment:
Why use this non-standard `[] dataframe` syntax?
```suggestion
#' @param input_dat A [data.frame] of modeled antibody kinetic parameter values.
#' @param original_data A [data.frame] of the original input dataset.
```